docs/examples/plot_linear_quickstart.py

```diff
@@ -52,9 +52,25 @@
 ######################################################################
 # ``preds`` holds the decision values, i.e. the raw values
 # outputted by the model. To transform it into predictions,
-# the simplest way is to take the positive values as
-# the labels predicted to be associated with the sample,
-# i.e. ``preds > 0``.
+# the simplest way is to take the positive values as the labels predicted
+# to be associated with the sample, i.e. ``preds > 0``. 
+
+preds[preds > 0] = 1
+preds[preds <= 0] = 0
```

Comment:
This should be `labelmask = preds > 0` so that `labelmask.dtype` is `bool`. `preds` is used later to compute P@K and should not be modified.